main/cron/fix_online_time.php

```diff
@@ -0,0 +1,99 @@
+<?php
+require_once __DIR__.'/../inc/global.inc.php';
+require_once api_get_path(LIBRARY_PATH).'database.lib.php';
```

Comment:
Don't need to include the database.lib.php file, is already called in global.inc.php